libs/ballot-interpreter/src/interpret_all_bubble_ballot.test.ts

```diff
@@ -0,0 +1,106 @@
+import { assert, Optional, range } from '@votingworks/basics';
+import { allBubbleBallotFixtures } from '@votingworks/hmpb-render-backend';
+import { Candidate, CandidateVote } from '@votingworks/types';
+import { singlePrecinctSelectionFor } from '@votingworks/utils';
+import {
+  ballotPdfToPageImages,
+  renderBallotToPageImages,
+  sortVotesDict,
+} from '../test/helpers/interpretation';
+import { interpretSheet } from './interpret';
+
+describe('Interpret - HMPB - All bubble ballot', () => {
+  const {
+    electionDefinition,
+    blankBallotPath,
+    filledBallotPath,
+    cyclingTestDeck,
+  } = allBubbleBallotFixtures;
+  const { election } = electionDefinition;
+  const precinctId = election.precincts[0]!.id;
+
+  const [frontContest, backContest] = election.contests;
+  assert(frontContest?.type === 'candidate');
+  assert(backContest?.type === 'candidate');
+
+  test('Blank ballot interpretation', async () => {
+    const ballotImagePaths = await ballotPdfToPageImages(blankBallotPath);
+    const [frontResult, backResult] = await interpretSheet(
+      {
+        electionDefinition,
+        precinctSelection: singlePrecinctSelectionFor(precinctId),
+        testMode: true,
+      },
+      ballotImagePaths
+    );
```

Comment:
This is basically the pattern of change that I made throughout this commit. The assertions are left unchanged